spec/examples/ex2-pop-on.xml

```diff
@@ -2,8 +2,8 @@
   xmlns:ttp="http://www.w3.org/ns/ttml#parameter" xmlns:tts="http://www.w3.org/ns/ttml#styling">
   <head>
     <layout>
-      <region xml:id="r1" tts:color="white" tts:origin="10c 4c" tts:extent="40c 1c"/>
-      <region xml:id="r2" tts:color="yellow" tts:origin="10c 8c" tts:extent="40c 1c"/>
+      <region xml:id="r1" tts:color="white" tts:origin="10c 4c" tts:extent="40c 1c" tts:displayAlign="after"/>
+      <region xml:id="r2" tts:color="yellow" tts:origin="10c 8c" tts:extent="40c 1c" tts:displayAlign="after"/>
```

Comment:
I don't think this is actively harmful but it is unnecessary - the region's extent is only `1c` high. Adding this detail could be confusing.